.orchestra/config/components/revng-vscode-web.yml

```diff
@@ -0,0 +1,70 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_vscode_web_args():
+binary_archives: private
+dependencies:
+- revng-c
+build_dependencies:
+- revng-ui
+- revng-theme
+- vscode-web
+configure: |
+  mkdir -p "$BUILD_DIR"
+install: |
+  PYTHON_PACKAGE_BASE=$("${ORCHESTRA_DOTDIR}/support/python-package-base")
```

Comment:
In order to go this route we need to:
1. Rename the script to `get-python-path` (or similar) and take an argument for the entry of `get_paths`.
2. Drop the `python_version` and `full_python_version` data values in `global_options.yml`.
3. Update `pip.lib.yml` to use `include` or `platinclude`

Reply:
Implemented as part of https://github.com/revng/orchestra/pull/103